brd-ios/breadwallet/src/Redesign/Scenes/Profile/ProfileViewController.swift

```diff
@@ -62,7 +62,8 @@ class ProfileViewController: BaseTableViewController<ProfileCoordinator,
                     (.levelTwo, .levelTwo(.levelTwo)):
                     config.status = Presets.VerificationView.verified.status
                     
-                case (.levelTwo, .levelTwo(.submitted)):
+                case (.levelOne, .emailPending),
```

Comment:
should this logic and the one on AccountVerificationViewController on line 65 be the same? or similar?

Reply:
similar but not the same.. those are different configs
![Screenshot 2022-11-04 at 08 12 38](https://user-images.githubusercontent.com/40755142/199917299-a862ad32-e845-4384-a027-f3e119a78e65.png)
![Screenshot 2022-11-04 at 08 13 36](https://user-images.githubusercontent.com/40755142/199917306-5d6c39ba-4a4f-45ec-8ce4-3d469e351d70.png)